app/models/user.py

```diff
@@ -78,4 +78,9 @@ def isProgramManagerFor(self, program):
     def isProgramManagerForEvent(self, event):
         # Looks to see who the Program Manager for a specific event is
         return self.isProgramManagerFor(event.program)
+    
+    def isProgramManager(self):
+        from app.models.programManager import ProgramManager
+        return ProgramManager.select().where(ProgramManager.user == self).exists()
```

Comment:
We should cache this result like we do in `isProgramManagerFor`, because otherwise we run a query every time we call the method